frontend/src/pages/dashboard/constants.ts

```diff
@@ -36,8 +36,8 @@ export enum DashboardColumns {
   STATUS = 'Status',
   SUPERVISOR_APPROVAL = 'Supervisor Approval',
   ICS = 'ICS',
-  DATE_APPLIED = 'Date Applied',
-  DATE_APPROVED = 'Date Approved',
+  DATE_APPLIED = 'Applied',
```

Comment:
This change seems like not a big deal, but was it approved?

Reply:
It was in the UI designs